app/api/serializers.py

```diff
@@ -31,3 +32,26 @@ class ItemSchemaSerializer(ModelSerializer):
     class Meta:
         model = ItemSchema
         read_only_fields = ('uuid',)
+
+
+class BoundarySerializer(GeoModelSerializer):
+
+    # TODO: How do we automagically detect that this is a nullable field and
+    #   set allow_null accordingly?
+    #   I guess we could just create a NullJsonBField instead?
+    errors = JsonBField(allow_null=True)
+
+    def __init__(self, *args, **kwargs):
+        """ Override init to delete some larger unecessary fields from the list endpoint """
+        super(BoundarySerializer, self).__init__(*args, **kwargs)
+        if len(args) > 0 and isinstance(args[0], Iterable):
```

Comment:
Hmm, I feel like it might be a bit more in keeping with the DRF's class hierarchy to subclass this serializer, set `exclude = ('geom')` in the subclass,and then figure out which Serializer to use in the Viewset's `get_serializer_class` method based on the type of request. Not a super huge deal though.